fusor-ember-cli/app/mixins/deployment-controller-mixin.js

```diff
@@ -162,19 +188,15 @@ export default Ember.Mixin.create({
           return 'review';
         }
       } else if (this.get('step2RouteName') === 'openstack') {
-        if (this.get('isCloudForms')) {
+        if (this.get('isOpenShift')) {
+          return 'openshift';
+        } else if (this.get('isCloudForms')) {
           return 'cloudforms';
         } else if (this.get('isSubscriptions')) {
           return 'subscriptions';
         } else {
           return 'review';
         }
-      } else if (this.get('step2RouteName') === 'cloudforms') {
```

Comment:
I removed the possibility that cloudforms can be step number 2 since RHEV or OSP needs to be selected.